src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IDeclarationExpression.cs

```diff
@@ -106,7 +107,7 @@ public void M2(out int i)
     Predecessors: [B4]
     Statements (0)
 ";
-            VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedFlowGraph, expectedDiagnostics);
+            VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedFlowGraph, expectedDiagnostics, parseOptions: TestOptions.Regular8);
```

Comment:
If this affects the output, please add verification for what it changes as well.